vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_uefi.rs

```diff
@@ -122,3 +124,40 @@ async fn auto_vtl2_range(config: PetriVmConfigOpenVmm) -> Result<(), anyhow::Err
 
     Ok(())
 }
+
+/// Boot OpenHCL, and validate that we did not see any numa errors from the
+/// kernel parsing the bootloader provided device tree.
+///
+/// TODO: OpenVMM doesn't support multiple numa nodes yet, but when it does, we
```

Comment:
Sure it does.

Reply:
Oh, but maybe only for processors, not for memory.

Reply:
hmm... i guess we didn't support it thru the cli

Reply:
yeah we don't support it for memory, which is what we'd need here.